legacy/testdata/rulesets.json

```diff
@@ -609,6 +609,63 @@
             }
         }
     },
+    {
+        "legacy_ruleset": {
+            "uuid": "bf8bcadf-7a26-4c4b-a5ab-577987ac3c20",
+            "x": 374,
+            "y": 668,
+            "label": "Selfie",
+            "rules": [
+                {
+                    "uuid": "b26f9640-93b4-409e-b65e-7d0ee1e886dd",
+                    "category": {
+                        "eng": "All Responses"
+                    },
+                    "destination": "5b977652-91e3-48be-8e86-7c8094b4aa8f",
+                    "destination_type": "A",
+                    "test": {
+                        "type": "true"
+                    },
+                    "label": null
+                }
+            ],
+            "finished_key": null,
+            "ruleset_type": "wait_photo",
+            "response_type": "",
+            "operand": "@step.value",
+            "config": {}
+        },
+        "collapse_exits": true,
+        "expected_node": {
+            "uuid": "bf8bcadf-7a26-4c4b-a5ab-577987ac3c20",
+            "router": {
+                "type": "switch",
+                "result_name": "Selfie",
+                "default_exit_uuid": "b26f9640-93b4-409e-b65e-7d0ee1e886dd",
+                "operand": "@input",
+                "cases": []
+            },
+            "wait": {
+                "type": "msg",
+                "media_hint": "image"
+            },
+            "exits": [
+                {
+                    "uuid": "b26f9640-93b4-409e-b65e-7d0ee1e886dd",
+                    "name": "All Responses",
+                    "destination_node_uuid": "5b977652-91e3-48be-8e86-7c8094b4aa8f"
+                }
+            ]
+        },
+        "expected_localization": {},
+        "expected_ui": {
+            "type": "wait_for_response",
```

Comment:
@ericnewcomer thoughts on what the UI part of this should be?

Reply:
Since the UI needs to be in the business of creating media_hints it seems letting it switch on those is sufficient.